docs/user_manual/components.md

```diff
@@ -863,10 +863,6 @@ $$
 
 ### Generic Current Sensor
 
-```{warning}
-At the time of writing, this feature is still experimental and is not yet publicly available.
```

Comment:
Might be good to mention something like: "At the time of writing, current sensors are only supported for Iterative Linear State Estimation and not yet for Newton Raphson State Estimation.